csharp/ql/src/semmle/code/csharp/ir/implementation/raw/internal/TranslatedExpr.qll

```diff
@@ -709,6 +711,24 @@ abstract class TranslatedTransparentExpr extends TranslatedNonConstantExpr {
   abstract TranslatedExpr getOperand();
 }
 
+class TranslatedPointerIndirectionExpr extends TranslatedPointerOps {
+  TranslatedPointerIndirectionExpr() {
+    // *p is the same as p until the result is loaded.
+    expr instanceof PointerIndirectionExpr
```

Comment:
Why not `override PointerIndirectionExpr expr`?

Reply:
Done